content/fr/edge-computing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Edge Computing
+status: Completed
+category: Technology
+---
+
+L'Edge Computing (informatique en périphérie) est une [architecture distribuée](/fr/distributed-systems/) qui déplace une partie de la capacité de stockage et de calcul du centre de données principal vers la source de données.
+Les données collectées sont traitées localement (par exemple, dans une usine, dans un magasin ou à travers toute une ville) plutôt que d'être envoyées à un centre de données centralisé pour traitement et analyse.
+Ces unités ou dispositifs de traitement locaux représentent la périphérie du système, tandis que le centre de données en est le centre.
+L'information calculée par ces unités périphériques est ensuite renvoyée au centre de données principal pour un traitement ultérieur.
+Parmi les exemples de Edge Computing, nous avons les bracelets connectés ("wrists gadgets"), ou les ordinateurs qui analysent le flux de circulation.
+
+## Problème auquel il répond
+
+Au cours de la dernière décennie, nous avons constaté une augmentation du nombre de dispositifs de périphérie de réseau (par exemple, téléphones mobiles, montres intelligentes ou capteurs).
+Dans certains cas, le traitement des données en temps réel n'est pas seulement un plus, mais vital.
+Pensez aux voitures autonomes.
+Maintenant, imaginez que les données des capteurs de la voiture doivent être transférées à un centre de données pour traitement avant d'être renvoyées au véhicule afin qu'il puisse réagir de manière appropriée.
+La latence inhérente au réseau pourrait être fatale.
+Bien que cela soit un exemple extrême, la plupart des utilisateurs ne voudraient pas utiliser un appareil intelligent incapable de fournir un retour d'information instantané.
+
+## Quel en est l'utilité
+
+Comme décrit ci-dessus, pour que les dispositifs de périphérie de réseau soient utiles, ils doivent effectuer au moins une partie du traitement et de l'analyse localement pour fournir un retour d'information quasi instantanée aux utilisateurs.
+Cela est réalisé en déplaçant certaines ressources de stockage et de traitement du centre de données vers l'endroit où les données sont générées : le dispositif de périphérie de réseau.
+Les données traitées et non traitées sont ensuite envoyées au centre de données pour un traitement et un stockage ultérieur.
```

Comment:
Je pense que `ultérieur` prend un "s" car ce sont les "le traitement et le stockage" qui sont considérés dans cette phrase.

Reply:
fix in https://github.com/cncf/glossary/pull/2888/commits/42ead3c4fa4972a08f6c8961c0cda9f34d291551